CMakeLists.txt

```diff
@@ -128,7 +128,7 @@ if(BUILD_COMMON_CURL)
     find_package(CURL REQUIRED)
     set(CURL_LIBRARIES CURL::libcurl)
   else()
-    pkg_check_modules(CURL REQUIRED libcurl)
+    find_package(CURL REQUIRED)
```

Comment:
what does this change do??

Reply:
This change is to make the cproducer to compile for alpine toolchain as well